labs/lab-4/server.py

```diff
@@ -52,24 +52,28 @@
     # LAB 4 --------------------------------------------------
 
     # # write a query to insert 3 records into the Courses table
-    # query_1 = """ """
+    # query_1 = """INSERT INTO COURSES (NAME, COURSEID, PROFESSORID)
```

Comment:
You have a syntax error here because the first half of your query is commented out resulting in mismatched quotes